Source/Core/Core/BootManager.cpp

```diff
@@ -95,6 +95,9 @@ struct ConfigCache
   float m_EmulationSpeed;
   float m_OCFactor;
   bool m_OCEnable;
+  bool m_RAMOverrideEnable;
```

Comment:
This isn't necessary if you use the new config system (the Common/Config and Core/Config stuff). Ditto for all the other additions to BootManager

Reply:
I don't understand what you're saying isn't necessary here.  I don't even know where the new config system is.

Reply:
The new config system is in Common/Config and Core/Config, as leoetlino mentioned. If you want to know how to add a new setting to it, you can use e.g. 3c9eb37 as a reference. (You only need to look at the changes in Core/Config.)

Reply:
Ooooooooh that's a folder you were referencing.  I was confused.  Alright, I'll look into this.

Reply:
Quick question, does the new config system have a cache to restore the "trampled on values" like the old system has?  I'm taking advantage of that in the fix for the FIFO player.

Reply:
Actually, it has something better: Layers. At boot, instead of overwriting the values in the base layer, the value changes are added to a different layer. The new config system takes both layers into account when you want to fetch the value of a setting (or not – it is possible to e.g. fetch the value of the base layer specifically). Then when emulation ends, the layer that was used for overriding settings for this emulation session is discarded, making those overridden values go back to normal.